components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -18,6 +18,21 @@
 #import "MaterialElevation.h"
 
 @class MDCActionSheetAction;
+@class MDCActionSheetController;
+
+/**
+ MDCActionSheetControllerDelegate provides a method that allows a delegate of an
```

Comment:
Nit: I don't think we typically use the `<#typeDeclaration>` within the docs for that `type`. 

_Suggestion:_
`A method for responding to dismissal of an @c MDCActionSheetController.`
**_Note:_** This is just a suggestion feel free to tweak it if you'd like, I'm just against using the type name in the explanation of the type.

Reply:
Done